charts/keda-http-operator/values.yaml

```
@@ -1,2 +1,4 @@
 version: v0.1.0-v1alpha1
-image: arschles/keda-http-operator
+image: ghcr.io/kedacore/http-add-on-operator
+interceptorImage: ghcr.io/kedacore/http-add-on-interceptor
+externalScalerImage: ghcr.io/kedacore/http-add-on-scaler
```

It's a nit, but it's best to use some structure and be more consistent:
```suggestion
images:
  operator: ghcr.io/kedacore/http-add-on-operator
  interceptor: ghcr.io/kedacore/http-add-on-interceptor
  externalScaler: ghcr.io/kedacore/http-add-on-scaler
```

You'll have to update all references though.